code/modules/ai/ai_behaviors/carbon/xeno/xeno_illusion.dm

```diff
@@ -0,0 +1,20 @@
+/datum/ai_behavior/carbon/xeno/illusion
```

Comment:
so illusions can open doors and locker. I could fix this, but is it a big issue? Your call